src/components/company.js

```diff
@@ -3,15 +3,15 @@ import React from 'react';
 export default function Company (props) {
     const { company = {} } = props;
     return (
-        <div className="Details">
+        <div className="DetailsCompany" name='company'>
```

Comment:
Please check that the `name` attribute can be applied for the set of dom elements
https://www.w3schools.com/tags/att_name.asp
there is no `div`.
But we don't need it, we can use class name to find this section.